requirements/base.txt

```diff
@@ -15,6 +15,7 @@ serverlessrepo==0.1.10
 aws_lambda_builders==1.19.0
 tomlkit==0.7.2
 watchdog==2.1.2
+pyopenssl==22.0.0
```

Comment:
Please run this step in PR description to update reproducable linux file;
> make update-reproducible-reqs

Reply:
Sure I will update the file